ui/raidboss/data/05-shb/dungeon/amaurot.js

```diff
@@ -129,9 +129,10 @@
         'Terminus Beholder': 'Terminus-Betrachter',
         'Mithridates': 'Mithridates',
         'Engage!': 'Start!',
-        'The First Doom': 'Erstes Unheil',
-        'The Second Doom': 'Zweites Unheil',
-        'The Third Doom': 'Drittes Unheil',
+        'The First Doom will be sealed off': 'Noch 15 Sekunden bis sich der Zugang zu Erstes Unheil schließt',
```

Comment:
Noch 15 Sekunden, bis sich der Zugang zum Ersten Unheil schließt.
Noch 15 Sekunden, bis sich der Zugang zum Zweiten Unheil schließt.
Noch 15 Sekunden, bis sich der Zugang zum Dritten Unheil schließt.

Again these are directly gathered from the logfiles